<launch>
  <arg name="color" default="yellow" />

  <group ns="ssl_joy">
    <param name="friend_color" type="string" value="$(arg color)" />
    
    <node name="joy_node" pkg="joy" type="joy_node">
      <param name="~dev" value="/dev/input/js0" />
    </node>

    <node name="ssl_joystick" pkg="ssl_joystick_operation" type="joystick_node.py" output="screen">
      <rosparam file="$(find ssl_joystick_operation)/param/ps4_config.yaml" />
    </node>
  </group>

</launch>
